src/mame/drivers/tim011.cpp

```diff
@@ -138,37 +156,38 @@ void tim011_state::tim011(machine_config &config)
 	HD64180RP(config, m_maincpu, XTAL(12'288'000)); // location U17 HD64180
 	m_maincpu->set_addrmap(AS_PROGRAM, &tim011_state::tim011_mem);
 	m_maincpu->set_addrmap(AS_IO, &tim011_state::tim011_io);
-	m_maincpu->set_vblank_int("screen", FUNC(tim011_state::irq0_line_hold));
+	m_maincpu->tend1_wr_callback().set([this](int state) { m_fdc->tc_w(state); });
```

Comment:
You’ve still got the trampoline – if you use `tc_line_w` you don’t need a trampoline.

Also, if you force push over the branch every time, it makes it harder to review because it hides what the actual changes are from one state to the next.

Reply:
Fixed this. Sorry for force push, wanted that part from Carl be under his name.